containers/base/Dockerfile

```diff
@@ -1,252 +1,293 @@
-# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Unclear on the change, but looks like all lines were modified? Can you fix so the delta is clearer?